### DISCUSSION TITLE
Using InkAnalyzer in WPF

### DISCUSSION THREAD

**Discussion by borioan** [dotnet/wpf#5895] [Q&A]
Hello!

Using libraries to detect geometric shapes using Ink Analyzer is a very useful tool. Is the Microsoft team considering introducing it for use as in the case of UWP? 
Thank you!

**Comment by vishalmsft** [comment]
Hello @borioan ,

Thank for the suggestion. Please note, this is WPF repository. UWP has own roadmap. 
Please try the discussion on [UWP repository](https://github.com/MicrosoftDocs/windows-uwp).

Thanks

**Reply by borioan** [reply]
Hello!
This question / suggestion is specific exactly for WPF. Btw I don't understand how this important feature has been excluded for years in development of WPF applications. Why? Why Ink Analyzer is just a UWP feature and is not a general one, included for all Windows technology?

**Reply by vishalmsft** [reply]
I am sorry, misunderstood the question. The Feature is part of UWP. 
Even [InkCanvas ](https://docs.microsoft.com/en-us/windows/communitytoolkit/controls/wpf-winforms/inkcanvas) is wrapper class.
